ui/app/adapters/job.js

```diff
@@ -68,4 +68,20 @@ export default class JobAdapter extends WatchableNamespaceIDs {
       },
     });
   }
+
+  scale(job, group, count, reason) {
+    const url = addToPath(this.urlForFindRecord(job.get('id'), 'job'), '/scale');
```

Comment:
I hadn’t known about `addToPath` before, I feel like I could have used it at some point, I’ll try to remember it exists 😯